src/editor.css

```diff
@@ -173,3 +173,7 @@
 .file-list-empty > div {
   margin: 10px 0;
 }
+
```

Comment:
Let's make the templates button container `overflow-x: auto` (and [this property for iOS, just in case](https://css-tricks.com/snippets/css/momentum-scrolling-on-ios-overflow-elements/)) in case the container doesn't fit all the previews.